ioctl/newcmd/action/action_test.go

```diff
@@ -270,3 +272,70 @@ func TestSendAction(t *testing.T) {
+
+	ks := keystore.NewKeyStore(t.TempDir(), 2, 1)
+	acc, err := ks.NewAccount("123456")
+	require.NoError(err)
+	accAddr, err := address.FromBytes(acc.Address.Bytes())
+	require.NoError(err)
+
+	chainMetaResponse := &iotexapi.GetChainMetaResponse{ChainMeta: &iotextypes.ChainMeta{}}
+	elp := createEnvelope(0)
+	cost, err := elp.Cost()
+	require.NoError(err)
+	accountResponse := &iotexapi.GetAccountResponse{AccountMeta: &iotextypes.AccountMeta{
+		Address:      accAddr.String(),
+		Nonce:        1,
+		PendingNonce: 1,
+		Balance:      cost.String(),
+	}}
+
+	client.EXPECT().SelectTranslation(gomock.Any()).Return("action", config.English).Times(4)
+	client.EXPECT().SetEndpointWithFlag(gomock.Any()).Do(func(_ func(*string, string, string, string)) {
+		// SetEndpointWithFlag take a function as parameter
+	}).Times(2)
+	client.EXPECT().SetInsecureWithFlag(gomock.Any()).Do(func(_ func(*bool, string, bool, string)) {
+		// SetInsecureWithFlag take a function as parameter
+	}).Times(2)
+	client.EXPECT().APIServiceClient().Return(apiServiceClient, nil).Times(2)
+
+	t.Run("reads smart contract on IoTeX blockchain", func(t *testing.T) {
+		client.EXPECT().AddressWithDefaultIfNotExist(gomock.Any()).Return("test", nil)
+		apiServiceClient.EXPECT().GetChainMeta(gomock.Any(), gomock.Any()).Return(chainMetaResponse, nil)
+		apiServiceClient.EXPECT().GetAccount(gomock.Any(), gomock.Any()).Return(accountResponse, nil)
+		apiServiceClient.EXPECT().SendAction(gomock.Any(), gomock.Any()).Return(&iotexapi.SendActionResponse{}, nil)
+		apiServiceClient.EXPECT().ReadContract(gomock.Any(), gomock.Any()).Return(&iotexapi.ReadContractResponse{
+			Data: "test",
+		}, nil)
+
+		cmd := NewActionCmd(client)
+		registerSignerFlag(client, cmd)
+		_, err := util.ExecuteCmd(cmd, _signerFlag, "test")
+		require.NoError(err)
+		signer, err := cmd.Flags().GetString(signerFlagLabel)
+		require.NoError(err)
```

Comment:
delete,  just want singer's value,  use `"test"` simply

Reply:
Delete line#253~255  #266~268  etc..  in `TestSendAction`.
Because just want the value of `passwd`